$OpenBSD$

Now j.root-servers.net is 192.158.128.30. (SERIAL 2002110501)
Use m.root-servers.net first for Japenese user.
Move a.root-servers.net to last.

--- dnsroots.global.orig	Wed Jul  9 12:45:23 2003
+++ dnsroots.global	Wed Jul  9 12:45:49 2003
@@ -1,4 +1,4 @@
-198.41.0.4
+202.12.27.33
 128.9.0.107
 192.33.4.12
 128.8.10.90
@@ -7,7 +7,7 @@
 192.112.36.4
 128.63.2.53
 192.36.148.17
-198.41.0.10
+192.58.128.30
 193.0.14.129
 198.32.64.12
-202.12.27.33
+198.41.0.4
